modules/java-modules/http/src/main/java/org/syslog_ng/http/HTTPDestination.java

```diff
@@ -62,12 +62,8 @@ public boolean init() {
         try {
             options.init();
         } catch (InvalidOptionException e) {
-            return false;
-        }
-        try {
-            this.url = new URL(options.getURL());
-        } catch (MalformedURLException e) {
-            logger.error("A properly formatted URL is a required option for this destination");
+            logger.error("error during init: InvalidOptionException.");
+            logger.error(e);
```

Comment:
AFAIK if you are using the `toString` method of the Exception (and implicitly you are using: it will be called when you pass the exception object to a print/log method), then the name of the `Exception` will be also printed which means that it is enough to keep the second `logger.error`.